pages/common/in2csv.md

```diff
@@ -0,0 +1,20 @@
+# in2csv
+
+> Converts various tabular data formats into CSV.
+> Included in csvkit.
+
+- Convert an XLS file to CSV:
+
+`in2csv {{data.xls}}`
+
+- Convert a DBF file to a CSV file:
+
+`in2csv {{data.dbf}} > {{data.csv}}`
+
+- Convert a specific sheet from an XLSX file to CSV:
+
+`in2csv --sheet={{sheet_name}} {{data.xlsx}}`
+
+- Fetch csvkit's open issues from GitHub's JSON API, and convert them to CSV:
```

Comment:
Actually I missed this one. That's a really cool oneliner and demonstrates the real usefulness of the tool but it's a little too much beyond the tool's core. I sometimes don't mind very practical examples but it's not completely in the tldr spirit, so I'll ask @igorshubovych to have a second look.